audit_report/3.1.0/src/00_09_introduction.rst

```diff
@@ -1,9 +1,267 @@
+`dilithium`, `dilithium_aes`, `dilithium_common`,
+`kyber`, `kyber_90s`, `kyber_common`,
+`sha1_armv8`, `sha1_sse2`, `sha1_x86`,
+`sphincsplus_common`, `sphincsplus_sha2`, `sphincsplus_shake`.
+Patches that don't alter any of the above-mentioned components or relevant
+modules are considered out-of-scope.
+
+Below is the full list of modules (from ``src/lib``) whose changes were
+reviewed:
+
+.. list-table::
+
+   * - aead
+     - aes
+     - aes_armv8
+     - aes_ni
+     - aes_power8
+   * - aes_vperm
+     - argon2
+     - argon2_avx2
+     - argon2_ssse3
+     - argon2fmt
+   * - asn1
+     - auto_rng
+     - base
+     - base64
+     - bigint
+   * - blake2
+     - block
+     - cbc
+     - ccm
+     - certstor
+   * - certstor_flatfile
+     - certstor_macos
+     - certstor_sql
+     - certstor_sqlite3
+     - certstor_system
+   * - certstor_system_macos
+     - certstor_system_windows
+     - certstor_windows
+     - cmac
+     - cpuid
+   * - ctr
+     - dh
+     - dilithium
+     - dilithium_aes
+     - dilithium_common
+   * - dl_algo
+     - dl_group
+     - dlies
+     - dsa
+     - dyn_load
+   * - ec_group
+     - ecc_key
+     - ecdh
+     - ecdsa
+     - ecgdsa
+   * - ecies
+     - eckcdsa
+     - eme_oaep
+     - eme_pkcs1
+     - emsa_pkcs1
+   * - emsa_pssr
+     - entropy
+     - ffi
+     - gcm
+     - getentropy
+   * - ghash
+     - ghash_cpu
+     - ghash_vperm
+     - gmac
+     - hash
+   * - hash_id
+     - hex
+     - hkdf
+     - hmac
+     - hmac_drbg
+   * - http_util
+     - iso9796
+     - kdf
+     - kdf1_iso18033
+     - keypair
+   * - kyber
+     - kyber_90s
+     - kyber_common
+     - locking_allocator
+     - mac
+   * - mdx_hash
+     - mem_pool
+     - mgf1
+     - mode_pad
+     - modes
+   * - mp
+     - numbertheory
+     - pbkdf
+     - pem
+     - pk_pad
+   * - pkcs11
+     - poly_dbl
+     - prf_tls
+     - processor_rng
+     - pubkey
+   * - rdseed
+     - rng
+     - rsa
+     - sha1
+     - sha1_armv8
+   * - sha1_sse2
+     - sha1_x86
+     - sha2_32
+     - sha2_32_armv8
+     - sha2_32_bmi2
+   * - sha2_32_x86
+     - sha2_64
+     - sha2_64_bmi2
+     - sha3
+     - sha3_bmi2
+   * - shake
+     - shake_cipher
+     - simd
+     - socket
+     - sp800_108
+   * - sp800_56c
+     - sphincsplus_common
+     - sphincsplus_sha2
+     - sphincsplus_shake
+     - stateful_rng
+   * - stream
+     - system_rng
+     - thread_utils
+     - tls
+     - tls_cbc
+   * - tls12
+     - tls13
+     - trunc_hash
+     - utils
+     - win32_stats
+   * - x509
+     - xmss
+     - xts
+     -
+     -
+
+The following previously existing modules are now in scope
+and were fully reviewed:
+
+- argon2, argon2_avx2, argon2_ssse3
+- blake2
+- hkdf
+- shake
+
+
+Patch Description Content
+-------------------------
+
+:ref:`Chapter 4<changes/additions>` shows the changes for this document's review iteration for all topics in scope.
```

Comment:
I wouldn't explicitly call this "Chapter 4". If we change things in the document structure, this might become bogus. Apparently, there are sphinx extensions that allow references by (dynamic) chapter number. Though, I'd suggest to just reword the sentence and use the referenced chapter's headline:

```suggestion
The changes for this document's review iteration for all relevant topics is found in :ref:`changes/additions`.
```

Reply:
To reference the actual chapter I needed a reference link for the auto generated chapter. I added it in 768e269 to the audit generator.